manager/controllers/app/m4dapplication_controller.go

```diff
@@ -222,34 +226,24 @@ func (r *M4DApplicationReconciler) reconcile(applicationContext *app.M4DApplicat
 	if hasError(applicationContext) {
 		return ctrl.Result{}, nil
 	}
-	r.Log.V(0).Info("Creating Blueprint")
```

Comment:
Log message will be wrong if plotter is in use

Reply:
Updated comments.